AdminOpsGuide/dbrepl.rst

```diff
@@ -942,19 +942,21 @@ Here is the code:
    export ydb_principal_editing=EDITING
    export ydb_routines="$PWD/$ydb_repl_instname $ydb_dist"
    #export ydb_routines="$PWD/$ydb_repl_instname $ydb_dist/libgtmutil.so"
-   # Here is an example of setting the ydb_routines environment variable:
-   # if [ -e  "$ydb_dist/libgtmutil.so" ] ; then export ydb_routines="$PWD/$ydb_repl_instname $ydb_dist/libgtmutil.so"
-   else export ydb_routines="$PWD/$ydb_repl_instname* $ydb_dist" ; fi
-   # For more examples on setting YottaDB related environment variables to reasonable values on POSIX shells, refer to the ydb_set_env script.
    #export LD_LIBRARY_PATH=/usr/local/lib
    #export ydb_crypt_config=$PWD/$ydb_repl_instname/config_file
    #echo -n "Enter Password for ydb_tls_passwd_${ydb_repl_instname}: ";export ydb_tls_passwd_${ydb_repl_instname}="`$ydb_dist/plugin/gtmcrypt/maskpass|tail -n 1|cut -f 3 -d " "`"
 
-Modify the env script according to your test environment. 
+.. note::
+   Here is an example of setting the ydb_routines environment variable:
+   if [ -e  "$ydb_dist/libgtmutil.so" ] ; then export ydb_routines="$PWD/$ydb_repl_instname $ydb_dist/libgtmutil.so"
+   else export ydb_routines="$PWD/$ydb_repl_instname* $ydb_dist" ; fi . For more examples on setting YottaDB related environment variables to reasonable values on POSIX shells, refer to the ydb_set_env script.
+
+
+Modify the ydbenv script according to your test environment. 
 
 **db_create**
 
-Creates a new sub-directory in the current directory, a global directory file with settings from gdemsr, and the YottaDB database file.
+Creates a new sub-directory in the current directory, a global directory file with settings taken from gdemsr, and the YottaDB database file.
 
 Here is the code:
```

Comment:
Check that the "Here is the code" sections match the actual script contents. I saw that at least the "originating_start" script differed.

Reply:
Made sure they matched - they had the same content, but some lines were in a different order.